File: api-python/src/vote.py
class Vote(object):
	def __init__(self, prenom, choix):
		self._prenom = prenom
		self._choix = choix

	def setPrenom(self, prenom):
		self._prenom = prenom

	def getPrenom(self):
		return self._prenom

	def setChoix(self, choix):
		self._choix = choix

	def getChoix(self):
		return self._choix	

File: leap-sender/VoteByLeapMotionProject/VoteByLeapMotionProject/Vote.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VoteByLeapMotionProject
{
  /// <summary>
  ///  class how permit the creation of object Vote
  ///  It's here then we find setters ans getters of Vote object. 
  /// </summary>
  class Vote
  {
    private String prenom;
    private Choix choixFait;

    public Vote(String prenom, Choix choixFait)
    {
      this.prenom = prenom;
      this.choixFait = choixFait;
    }
    
    public void setPrenom(String prenom)
    {
      this.prenom = prenom;
    }

    public String getPrenom()
    {
      return prenom;
    }
     
    public void setChoixFait(Choix choixFait)
    {
      this.choixFait = choixFait;
    }

    public Choix getChoixFait()
    {
      return choixFait;
    }
  }
}
